seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/client/HbaseClient.java

```diff
@@ -77,7 +80,31 @@ private HbaseClient(Connection connection, HbaseParameters hbaseParameters) {
                                             hbaseParameters.getNamespace(),
                                             hbaseParameters.getTable()))
                             .pool(HTable.getDefaultExecutor(hbaseConfiguration))
-                            .writeBufferSize(hbaseParameters.getWriteBufferSize());
+                            .writeBufferSize(hbaseParameters.getWriteBufferSize())
+                            .listener(
+                                    (e, mutator) -> {
+                                        for (int i = 0; i < e.getNumExceptions(); i++) {
```

Comment:
what's happend when all retry failed?

Reply:
> what's happend when all retry failed?

It is indeed a bug, and I haven't thought of a good way to prevent this situation from occurring. Moreover, there is currently no relevant log information when HBase write failures happen.

Reply:
we should handle this situation before merge this PR.

Reply:
he HBase client itself will also perform a certain number of retries.